### DISCUSSION TITLE
symbolically show the constraints

### DISCUSSION THREAD

**Discussion by styw1234** [yalmip/YALMIP#1409] [General discussion]
x=sdpvar(1,2)
y=2*sum(x)

As is known to all, we can use 'sdisplay' to symbolically show the unknown variable y with 'y=2*(x(1)+x(2))'
Then how to symbolically show the constraints?

**Comment by johanlofberg** [comment]
Unclear what you are talking about as you haven't defined any constraint, you've simply done an assignment. If you sdisplay(y) it will show 2*x(1)+2*x(2) as expected

**Reply by styw1234** [reply]
The constraints are denoted as follows.
x=sdpvar(1,2)
y=2*sum(x)
con=[];
con=[con,0<=x<=3,1<=sum(x)<=5]
how to display the constraints as
x(1)+x(2)<=5
-x(1)-x(2)<=-1
x(1)<=3
-x(1)<=0
x(2)<=3
-x(2)<=0

**Reply by johanlofberg** [reply]
the closest you can do is sdisplay(sdpvar(con))

**Reply by styw1234** [reply]
OK! Thank you very much!
